app/main.cjsx

```diff
@@ -47,6 +47,7 @@ routes = <Route handler={App}>
     <DefaultRoute name="user-profile" handler={require './pages/profile/feed'} />
     <Route name="user-profile-private-message" path="message" handler={require './pages/profile/private-message'} />
     <Route name="user-profile-stats" path="stats" handler={require './pages/profile/stats'} />
+    <Route name="moderations" handler={require './talk/moderations'} />
```

Comment:
I kinda took profiles to be a home for anything relating to an individual user. Is there another place you prefer for it to live?